stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/controller/ChannelController.kt

```diff
@@ -84,7 +85,11 @@ public sealed interface ChannelController {
     public val recoveryNeeded: Boolean
     public val cid: String
 
-    @Deprecated("Use offlineChannelData instead of it", replaceWith = ReplaceWith("ChannelController::offlineChannelData"))
+    @Deprecated(
+        level = DeprecationLevel.ERROR,
+        message = "Use offlineChannelData instead of it",
+        replaceWith = ReplaceWith("ChannelController::offlineChannelData"),
+    )
     public val channelData: LiveData<ChannelData>
 
     public val offlineChannelData: LiveData<OfflineChannelData>
```

Comment:
Are we going to rename this to `channelData` again after the method above is removed? The name is a bit confusing, looks like channels obtained only from the database.

Reply:
@ogkuzmin what was the plan?

Reply:
It's doubling. Since we expose live data ChannelData and have the same in flow version. We can't just delete it, that's why I had to deprecate old and expose the same from offline package with a little changed name

Reply:
we can consider changing its name after we delete the old `channelData` but hopefully, we'll get rid of that at some point